crawl-ref/source/ng-init.cc

```diff
@@ -2,7 +2,7 @@
  * @file
  * @brief Initializing non-player-related parts of a new game.
 **/
-/* TODO: 'you' shouldn't occur here.
+/* TODO: 'you' shouldn't occur here. (Except for Initiate?)
```

Comment:
update role name

Reply:
Good catch! Actually this line should just be deleted - it referred to the Temple location setting that Delver doesn't do.